### DISCUSSION TITLE
Introducing Projects on GitHub Mobile! (public beta)

### DISCUSSION THREAD

**Discussion by candyho** [community/community#35791] [Mobile]
<img width="1200" alt="188992932-ee7ee343-c1b8-46d7-b0c3-cf79337aecfb (1)" src="https://user-images.githubusercontent.com/7498102/195095187-72713259-a9f9-42cd-89e7-36d4dbfaeba3.png">

👋🏼Hey everyone,

We're excited to introduce GitHub Projects support on GitHub Mobile, now in public beta! You can now view and manage projects on the go. We would love for you to try it out and give us your early feedback here -  Do you find the feature useful? What else do you want us to support and why?

## How do I test this
If you are already a beta user - simply update to the latest beta builds through TestFlight for iOS or Google Play Store for Android.

If you are not a beta user yet, join GitHub Mobile beta program through the links below:

[Join iOS TestFlight](https://testflight.apple.com/join/NLskzwi5)
[Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)

## Feature highlights

- Access your projects through repo, organization, and user profiles
- Search for your desired projects
- View all of your project items 
- View items as grouped and configured on web
- All custom fields attached to an item are represented as metadata pill
- Long press on an item to perform the following quick actions:
  - Edit title
  - Edit custom fields
  - Close an issue
  - Delete an item from project
  - Share a project
- Tap on the project header to switch to other project views

## Known issues
Large projects might take longer or fail to load, we are working on optimizing the performance of larger projects. Please stay tuned for more updates.

For more details please refer to this [blog post](https://github.blog/2022-10-11-on-the-go-with-github-projects-on-github-mobile-public-beta/).

Happy testing! ❤️

**Comment by sammarks** [comment]
Just downloaded and tested; looking great! My only suggestion would be to allow bookmarking a project either via Favorites or Shortcuts on the home screen. It's a little cumbersome to navigate into an organization -> projects -> (the project) right now and would be more convenient to just tap an option on the home screen.

_If I left my feedback in the wrong spot, let me know and I'll move it!_

**Reply by candyho** [reply]
You left your feedback in the correct spot indeed! Thanks for sharing your feedback with us. We are glad that you like the feature.❤️

> My only suggestion would be to allow bookmarking a project either via Favorites or Shortcuts on the home screen. It's a little cumbersome to navigate into an organization -> projects -> (the project) right now and would be more convenient to just tap an option on the home screen.

Totally! This is something planned so please stay tuned for more updates.

**Reply by Nirlah** [reply]
Came here to say the same thing – for my personal workflow having a project on reach is the highest priority.
That being said, I'm happy the new projects made its way to mobile, I've been waiting for it!

**Reply by SUNSOL87** [reply]
Well dissaved ideas... Am really interested

**Comment by IgorKowalczyk** [comment]
I downloaded the update and tested it, looks and works great ❤️ 
Although it would be useful to have text if something doesn't have a description as it is in the browser version

<details>
 <summary>Desktop version [have alternative description]</summary>
<img src="https://user-images.githubusercontent.com/49127376/195147750-97a7210c-0207-4d5d-b854-dc9516a01e7f.png"/>
</details>


<details>
 <summary>Desktop version [no alternative text]</summary>
<img src="https://user-images.githubusercontent.com/49127376/195147916-d7223eeb-2719-4d15-9213-6d5e7d1a6af1.png"/>
</details>

**Reply by mxie** [reply]
@IgorKowalczyk 👋 Are you referring to the `No description provided` text?

**Reply by IgorKowalczyk** [reply]
yes that's what i talked about

**Reply by hano-1212** [reply]
> I downloaded the update and tested it, looks and works great ❤️ 
> 
> Although it would be useful to have text if something doesn't have a description as it is in the browser version
> 
> 
> 
> <details>
> 
>  <summary>Desktop version [have alternative description]</summary>
> 
> <img src="https://user-images.githubusercontent.com/49127376/195147750-97a7210c-0207-4d5d-b854-dc9516a01e7f.png"/>
> 
> </details>
> 
> 
> 
> 
> 
> <details>
> 
>  <summary>Desktop version [no alternative text]</summary>
> 
> <img src="https://user-images.githubusercontent.com/49127376/195147916-d7223eeb-2719-4d15-9213-6d5e7d1a6af1.png"/>
> 
> </details>
> 
> 
> 
>

**Reply by revvv289** [reply]
[]()

**Reply by duarteoficialhp** [reply]
Javad

**Comment by alexanderniebuhr** [comment]
one quick note I have, it seems that board views with cycles seem to fail loading even for small projects.

**Reply by mxie** [reply]
I'm sorry to hear that @alexanderniebuhr!

Do you have a link you can share with us to debug? Can you also tell us whether you're using the Android or iOS app? Thanks!

**Reply by alexanderniebuhr** [reply]
iOS app. The project is closed/private source. I could see if can replicate that behaviour.

**Comment by mbrevda** [comment]
I'm on beta (android) but don't see projects anywhere. Where can I find them?

**Reply by candyho** [reply]
Hey! You can access your projects via the repository, organization or your own user profile view.

**Reply by mbrevda** [reply]
> Hey! You can access your projects via the repository, organization or your own user profile view. 

Here is an org - what am I missing??

<img src="https://user-images.githubusercontent.com/293004/195333838-d131aff7-5c4a-474b-a7de-b558785ff4f1.png" alt="drawing" width="200"/>

**Reply by candyho** [reply]
Hmm can you try to access this public repo and see if you can see the project section in there? - https://github.com/heroku

**Reply by kapple19** [reply]
I can see the Heroku project in the app. But I don't see any way to create a new project for my personal profile or repository. Can mobile only view existing projects?

On a different note, I look forwared to boards being supported on mobile!

**Comment by hippyaki** [comment]
This could be weird. 🤢 But what if I could edit my code from the Mobile? 🤷‍♀️ There are few changes that strikes to me when I don't have my computer nearby. 😢

I wish I had the option to write the code from my Mobile for quick updates. Helps with updating changes requested during PR as well. 😆

I am open to criticism, on why it would be a bad idea as well. Educate me on this. 😉

**Reply by IgorKowalczyk** [reply]
https://github.blog/changelog/2022-09-13-github-for-ios-edit-files-in-pull-requests/

**Reply by hippyaki** [reply]
Is this coming p for Android too? And what about the editing directly from the repo?

**Reply by IgorKowalczyk** [reply]
Github announced that it will add PR editions on Android later this year meaning probably in the next few months. Editing directly in the repository has not yet been announced

**Reply by candyho** [reply]
> Is this coming p for Android too? And what about the editing directly from the repo?

Yes! this is planned as well 😉

**Reply by Solo3355** [reply]
Well i as for now i don't have my computer so i decided to use mobile

**Comment by j5155** [comment]
Unless I'm missing something, it seems like there's no way to add items to boards on mobile right now.
Looking at both my personal projects and ones from an organization I'm admin of on TestFlight version 1.85.0 and iOS 16.1, I'm not seeing any way of adding them. It would be very convenient to be able to entirely manage boards from mobile, especially for using boards for non-programming-related things on our team. 
I'm mainly asking in case I just missed the button, I would completely understand if this just hasn't been implemented yet.

**Reply by candyho** [reply]
Hey, thank you for your feedback! You haven't missed a button. Adding new items to a project isn't supported on mobile just yet, but it is a planned feature so please stay tuned for more updates!

**Reply by lmj0011** [reply]
I'm looking forward to it, I'm pretty amazed that the beta was released without that feature at least. I really have no reason to use this yet until that's added.

**Reply by bewuethr** [reply]
I'd love to have that, too! It _almost_ works on the mobile website, though I'm unable to submit after entering the title of a new issue 🙁

**Reply by L-U-C-K-Y** [reply]
Thanks for this, this is the only important missing feature for me.

**Reply by Frosinda** [reply]
Hello everyone and good morning!  I want to make my first comment because I just registered as a beginner I need to learn

**Reply by jeffvestal** [reply]
Agree with the others, this is super important. Once I can add items it will become useful and I'd love to use it to replace other tools on my phone/tablet.

**Reply by nickevansuk** [reply]
Agree with this - adding "Draft" items to the project board is such a helpful feature on web - especially when someone in the team wants to capture a thought or question but doesn't know which repo it should live in (for an org-level project). It would be really great to have it on GitHub Mobile.

There's currently a bug on the GitHub Projects web UI at the moment where you can't add new items (drafts) to a "Board" in Safari on iOS - so currently there is no way for a mobile user to add drafts to a "Board" view. Workaround is to add items via a list.

**Reply by rafaelurben** [reply]
Totally agree! Adding and managing draft items is one of the most useful features that are missing on GitHub Mobile.

**Reply by nickevansuk** [reply]
Agreed!

**Reply by sernaferna** [reply]
The last update on this thread was months ago but there still doesn't seem to be the ability to add an item. This is the only reason I'd want to use the mobile app; when I'm actively developing I have the "real" app open anyway; when I'm on the go is when I'd want to add something to my to-do list. 

Is this feature still on the roadmap?

**Reply by ChandlerLutz** [reply]
Agreed! For me this is the only feature I want on mobile -- the ability to add draft items on the go so I don't have to store them in my head. Thanks!

**Reply by rafaelurben** [reply]
@candyho Is there any news on this? Without adding or editing draft items, project support on mobile is kind of useless.

**Reply by Nabu-thinker-ru** [reply]
Year passed :( Where is the Add button?

**Reply by andystumpp** [reply]
The app would be a a LOT more valuable if there is a way to add items to my project from the mobile app. Can we get support for this?

**Reply by ryanlegler** [reply]
Too busy putting AI in all the things to finish this app I guess

**Reply by juanrh** [reply]
The app is quite useless as it is

**Reply by jkhereford** [reply]
+1, adding tasks on the go is actually the most important use case here for me.  Almost to the point of being the only use case.  Sort of surprised this one hasn't made the P0 list.

**Reply by davide-bleggi** [reply]
Completely agree. It's not so useful right now. Can't wait to see it.

**Reply by urfx** [reply]
need to add items to projects and then later sometimes change items to issues for repos... any news?

**Reply by urfx** [reply]
just a note that you can add issues to projects through setting up automation... and the app did just let me add an issue. but yes not "items"

**Reply by djsavvy** [reply]
Unless I'm missing something, this doesn't seem to be possible as of March 2025 :(

**Comment by tayv** [comment]
Projects on mobile looks great! Some minor suggestions for the iOS app:

### Idea 1:
- Persist the project's collapsible bucket state so that I can quickly find the bucket I use the most without having to scroll or re-collapse the buckets each time I re-open the project.

How it currently works:
- Each kanban lane/bucket resets to an expanded state after I navigate away from the project.

****Notes:**** 
- The left hand column on the kanban board is not always the most important bucket to view but on mobile this column lives at the top of the list and I have to scroll past it or manually collapse it to get to the bucket that's most important to me. This feels tedious.

---

### Idea 2
- Visually differentiate a bucket's title styling so that it's easier to spot headings when quickly scanning through the project list.

How it currently works:
- On iOS the background styling matches the rest of the list. 

Notes:
- From the blog post it looks like the android version already addresses this problem by using a grey background for bucket titles. It'd be great if this was matched in iOS.
- <img width="300" alt="android_buckettitle_styling" src="https://user-images.githubusercontent.com/48400779/199131008-c2327514-8987-4535-b377-41dbed6a8e52.png">

**Reply by candyho** [reply]
Hey Thanks for sharing these useful feedback! I have shared these with our internal team and we will look into it soon!

**Reply by Sxsky81** [reply]
- - - [ ] ****[_@_](https://tx.botdev.me/TRX/831697d38822894842c662c4d784da2aa48688ac3a34502629873c32e778c8ae)

**Comment by 66667888sssttt** [comment]
[]()

**Reply by guylepage3** [reply]
Glad to see someone working on this. I don't use the beta projects because it's so limiting. When will you support the classic projects?

**Comment by zmyznu** [comment]
I'm from China. Is there a way to get the build version of the test version directly without accessing Google Play? Thank you, because I can't access Google Play. Thank you

**Comment by SRAGEIN** [comment]
I want to learn how to be in control of apps like Facebook and messenger any apps I use I want to be in master control and

**Comment by fboucher** [comment]
Great App! I tried it on Android.
I couldn't approved PR without being assign to it... Not sure if it's by design... ;)

**Comment by Axaaa20** [comment]


**Comment by SUNSOL87** [comment]
Please, how extectly they make used GitHub for generating money ???

**Comment by nickevansuk** [comment]
We're just looking at putting the new GitHub Projects at the centre of our workflow (great product by the way!)

However Public Beta on iOS is full - is there any chance of more spaces being released? We're likely to be heavy users so would no doubt have feedback to offer - or otherwise (if it's too late for feedback and you're just trying to ship it), we'll certainly find any niggling bugs left over.

Thanks so much!

**Reply by candyho** [reply]
Hey there! Sorry for delayed response. We have actually done some cleaning to release more Public Beta seats on iOS last week! You should be able to join our Beta now! Looking forward to your feedback for Projects on Mobile. Just a heads up, we are working on optimizing the Projects UI/UX on mobile so you all should see a new look pretty soon after joining the Beta this week. :) 

Join iOS TestFlight
https://testflight.apple.com/join/NLskzwi5

**Reply by nickevansuk** [reply]
That's great, thanks! Have just joined.

As noted elsewhere it doesn't seem possible to add new draft items to the project via the app, move items between columns, or edit descriptions of draft items. Keen to make use of this, but have limited use cases for a read-only view (most or our interactions with projects involve updating something)

Will look out for the new UI/UX, thanks!

**Reply by candyho** [reply]
The ability to add a draft issue or an existing issue to a project is something we will work on next! Thanks for your feedback.

Regarding moving a project item from one column to another - you can actually do that on mobile! Try long press an item and choose "edit field", you should be able to edit the fields and the item will be moved to the grouped bucket accordingly.

You should be able to edit the title and the field of draft issue too!

**Reply by nickevansuk** [reply]
Ah yes, that wasn't obvious - thanks! Looks like you can't move the item up and down within the column though?

Ok great looking forward to editing coming out.

One additional bit of feedback - I use various other project management apps for different things (Todoist, Asana, Trello, Clickup, Sunsama), and in all cases it is only a few taps from the home screen to get to the project or board I'm looking for. In GitHub the projects are fairly deeply nested so I have to do quite a bit of navigation each time I open the app. It would be really helpful to be able to add shortcuts to a particular project to the Home screen in iOS - if I spend all day in a particular project board it's really helpful to be able to jump straight to it

**Comment by vLuckyyy** [comment]
Add tags for repositories

**Reply by daboujohan-hub** [reply]
Salut

**Comment by Woshimbo** [comment]
I can view the projects from the Profile page but after that I can't do anything with them. Can only Share them. I'm looking forward to Pages though.

**Comment by goran-revops** [comment]
Having everything in one place on GitHub is so convenient and streamlines the project management process. It's great to see that GitHub is becoming the one platform that developers and project managers need for the entire software development cycle.

**Comment by Nader822** [comment]
Nader

**Comment by CntrlX** [comment]
Hi just tested the app and I think there's more to come but it would be great if i could just add more items. Because it would help me write down my ideas and extra features on the go and help me remember it and implement it in the code.

And moreover i would like this app to be more like a GitHub companion rather than github itself.

**Comment by Amejonah1200** [comment]
Hello o/
I updated the app and found this feature. But you forgot one thing: editing the entry itself (if it's a draft)... how could you :c
It's this 🤏 this close to be perfect. I hope you add it soon. . _.
(I'm on beta right now)

**Comment by morgzz** [comment]
I won't use this feature if “Draft” verbiage is visible. It takes up to much real estate, it’s visually cringe and repetitive. Use a glyph instead or better yet not at all.

**Comment by 89257919619** [comment]
Svetlana sergeevna

**Comment by 89257919619** [comment]
Svetlana sergeevna

**Comment by 89257919619** [comment]
89261817383

**Comment by 89257919619** [comment]
> 89261817383

Светлана

**Comment by itslazyvar** [comment]
Not sure if this is still actively monitored, nevertheless here’s some feedback on the mobile app (on iOS):

### Option to create drafts

Like many have already said, we really need an option to create drafts to capture tasks that come up on the go. 

### Better state binding

Currently when a change is made on an item, this change is not immediately reflected in the view.   

Instead we have to back out of the projects section of the app entirely (not just out of the specific project in question), to be able to see the changes we made. 

Presumably because this triggers a new API fetch, but it would be better if this is properly managed with a binding that updates the view the moment something is changed. 

### Option to select an assignee

This speaks for itself I think. Currently there’s no way to select an assignee amongst the editing options. 

All the above is based on the app experience on iOS and the latest TestFlight build Version: 1.123.0 (105244228).

**Reply by yassine-cherni** [reply]
Yes

**Comment by fortis7789** [comment]
> <img width="1200" alt="188992932-ee7ee343-c1b8-46d7-b0c3-cf79337aecfb (1)" src="https://user-images.githubusercontent.com/7498102/195095187-72713259-a9f9-42cd-89e7-36d4dbfaeba3.png">
> 
> 👋🏼Hey everyone,
> 
> We're excited to introduce GitHub Projects support on GitHub Mobile, now in public beta! You can now view and manage projects on the go. We would love for you to try it out and give us your early feedback here -  Do you find the feature useful? What else do you want us to support and why?
> 
> ## How do I test this
> If you are already a beta user - simply update to the latest beta builds through TestFlight for iOS or Google Play Store for Android.
> 
> If you are not a beta user yet, join GitHub Mobile beta program through the links below:
> 
> [Join iOS TestFlight](https://testflight.apple.com/join/NLskzwi5)
> [Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)
> 
> ## Feature highlights
> 
> - Access your projects through repo, organization, and user profiles
> - Search for your desired projects
> - View all of your project items 
> - View items as grouped and configured on web
> - All custom fields attached to an item are represented as metadata pill
> - Long press on an item to perform the following quick actions:
>   - Edit title
>   - Edit custom fields
>   - Close an issue
>   - Delete an item from project
>   - Share a project
> - Tap on the project header to switch to other project views
> 
> ## Known issues
> Large projects might take longer or fail to load, we are working on optimizing the performance of larger projects. Please stay tuned for more updates.
> 
> For more details please refer to this [blog post](https://github.blog/2022-10-11-on-the-go-with-github-projects-on-github-mobile-public-beta/).
> 
> Happy testing! ❤️

**Reply by Amejonah1200** [reply]
Thank you for copy pasting... for what?

**Comment by lukepighetti** [comment]
Is there any way to create a new ticket draft in GitHub Projects on mobile? I can't seem to find any way to do this. 

I appear to be blocked on both the web client and the iOS app

**Reply by ArslanAmeer** [reply]
What I did or found is that you can always create an issue from the GitHub app home screen. When creating an issue, you can choose the project in which you would like to have your ticket. However, it is not possible to create draft.

**Reply by itslazyvar** [reply]
> What I did or found is that you can always create an issue from the GitHub app home screen. When creating an issue, you can choose the project in which you would like to have your ticket. However, it is not possible to create draft.

Thanks for sharing that!

That's actually a nice workaround I wasn't aware of, which for the most part does the trick for me. 

Still, it would be nice if GH could spare some attention to Projects on mobile, because it's starting to feel that it (and this thread) has been abandoned.

**Comment by Amirvulcan** [comment]
- [ ]طدف د# ز۵۵لبواج۰د

**Comment by Amirvulcan** [comment]
- [ ]طدف د# ز۵۵لبواج۰ر

**Comment by devil-tech007** [comment]
Cool stuff, grate job.

**Reply by duarteoficialhp** [reply]
[]()‎09926764324

**Comment by Ppaypunefon** [comment]
Very good

**Comment by furby-tm** [comment]
I used to have access to the beta but it's now full, any chance you can get expand some available slots in your public beta?

Mostly I'm just anticipating GitHub actions loading properly on iOS, it's been very intermittent.

**Comment by diveshthapa** [comment]
Yes, it is working 🥳🎇 for some project file but not all forms like table, list and also long projects types. It is available to country wise or every beta users?

**Reply by candyho** [reply]
Hey, the feature is already available for every users on the production build.

**Comment by xxprzemoxx** [comment]
co ma robić by dostać wypłatę moich środków...

wt., 1 paź 2024 o 14:36 Przemyslaw Karczewski ***@***.***>
napisał(a):

**Comment by Pointlesskey** [comment]
Product feedback

**Comment by itslazyvar** [comment]
@candyho Clearly the app is still receiving love and updates, but communication and engagement from GH on the app has been absent. 

What is the community to make of this? 

Does this mean it's not a priority anymore or that the project is frozen and in maintenance mode?

Does GH still want feedback and enhancement requests? What is the roadmap for the app?

**Reply by candyho** [reply]
Hey, thank you for reaching out and for sharing your concerns. We understand how important communication and transparency are, and we sincerely apologize if it feels like we haven’t been as engaged with the community as we should be.

We understand the importance of being able to create a draft issue and add it to a project on the go. We've already logged this request in our internal repo and will be reviewing it regularly to prioritize accordingly, along with other requests.

Thank you again for your passion and patience – your input helps us make the product better for everyone.

**Reply by itslazyvar** [reply]
Thank you for taking the time to reply. 
I'm sure this is not the only project that requires your attention, but it would be nice if we can get an update from time to time or some idea of what's in the in pipeline.

**Comment by olivermarshall2** [comment]
Hi. 
Am I right in thinking that you can't yet add an Item to a Board of a Project in the Beta Github app (on Android)? It's a shame as I was hoping I could add items on the go for my project rather than use Trello and then manually enter them from Trello to Github Projects later on. 

Hopefully you'll tell me a being a divot and the function is hidden somewhere in the UI :)

Olly

**Reply by candyho** [reply]
Hey you could go to any existing issue, and then tap on the `i` button at the bottom right to add that issue to a project of your chioce.

If you wanna create a new issue and add that to a project on the go, just tap on `+` button on the home screen and create a new issue, throughout the creation flow you should find an option to add that issue to a project by tapping on the "action pills" above the keyboard.

**Reply by leftweb** [reply]
Very helpful, as a _hopefully_ only interim solution.  Thank you!

**Reply by chinthaka-dinadasa** [reply]
Is this available for IOS as well @candyho ?

**Comment by kapple19** [comment]
I think what would ease people's minds re this project is... a publicly viewable GitHub Project for it 😛 where we can witness the tasks progressing.

**Comment by Guess440** [comment]
> <img width="1200" alt="188992932-ee7ee343-c1b8-46d7-b0c3-cf79337aecfb (1)" src="https://user-images.githubusercontent.com/7498102/195095187-72713259-a9f9-42cd-89e7-36d4dbfaeba3.png">
> 
> 👋🏼Hey everyone,
> 
> We're excited to introduce GitHub Projects support on GitHub Mobile, now in public beta! You can now view and manage projects on the go. We would love for you to try it out and give us your early feedback here -  Do you find the feature useful? What else do you want us to support and why?
> 
> ## How do I test this
> If you are already a beta user - simply update to the latest beta builds through TestFlight for iOS or Google Play Store for Android.
> 
> If you are not a beta user yet, join GitHub Mobile beta program through the links below:
> 
> [Join iOS TestFlight](https://testflight.apple.com/join/NLskzwi5)
> [Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)
> 
> ## Feature highlights
> 
> - Access your projects through repo, organization, and user profiles
> - Search for your desired projects
> - View all of your project items 
> - View items as grouped and configured on web
> - All custom fields attached to an item are represented as metadata pill
> - Long press on an item to perform the following quick actions:
>   - Edit title
>   - Edit custom fields
>   - Close an issue
>   - Delete an item from project
>   - Share a project
> - Tap on the project header to switch to other project views
> 
> ## Known issues
> Large projects might take longer or fail to load, we are working on optimizing the performance of larger projects. Please stay tuned for more updates.
> 
> For more details please refer to this [blog post](https://github.blog/2022-10-11-on-the-go-with-github-projects-on-github-mobile-public-beta/).
> 
> Happy testing! ❤️

cf6aca1d98828d9b07636e8cfdd37c654c529205

**Comment by 1q2w3e4r5t6z7u8** [comment]
😀

**Comment by VoxelPrismatic** [comment]
When can we expect to be able to add and manage tasks? Web UI is poor on mobile, and I like being able to write down my ideas on the go.

**Comment by hageldave** [comment]
Why is this still beta? This feature is 3 years old. Entire software is written in less time.

**Comment by vcscroll** [comment]
Up

**Comment by louisfoytikif200x6** [comment]
> <img width="1200" alt="188992932-ee7ee343-c1b8-46d7-b0c3-cf79337aecfb (1)" src="https://user-images.githubusercontent.com/7498102/195095187-72713259-a9f9-42cd-89e7-36d4dbfaeba3.png">
> 
> 👋🏼Hey everyone,
> 
> We're excited to introduce GitHub Projects support on GitHub Mobile, now in public beta! You can now view and manage projects on the go. We would love for you to try it out and give us your early feedback here -  Do you find the feature useful? What else do you want us to support and why?
> 
> ## How do I test this
> If you are already a beta user - simply update to the latest beta builds through TestFlight for iOS or Google Play Store for Android.
> 
> If you are not a beta user yet, join GitHub Mobile beta program through the links below:
> 
> [Join iOS TestFlight](https://testflight.apple.com/join/NLskzwi5)
> [Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)
> 
> ## Feature highlights
> 
> - Access your projects through repo, organization, and user profiles
> - Search for your desired projects
> - View all of your project items 
> - View items as grouped and configured on web
> - All custom fields attached to an item are represented as metadata pill
> - Long press on an item to perform the following quick actions:
>   - Edit title
>   - Edit custom fields
>   - Close an issue
>   - Delete an item from project
>   - Share a project
> - Tap on the project header to switch to other project views
> 
> ## Known issues
> Large projects might take longer or fail to load, we are working on optimizing the performance of larger projects. Please stay tuned for more updates.
> 
> For more details please refer to this [blog post](https://github.blog/2022-10-11-on-the-go-with-github-projects-on-github-mobile-public-beta/).
> 
> Happy testing! ❤️

**Reply by louisfoytikif200x6** [reply]
com.mempool.space
